recipes/xyz2mol/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "xyz2mol" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jensengroup/xyz2mol/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 81bd53319a5c3e489db330c6db61e0ac0424b2e530de7f9c0c87291675ef6031
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

This should fix Windows:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  entry_points:
    - xyz2mol=xyz2mol:main
```

---

I wasn't aware of that trick. Thanks!